iOS/README.md

```diff
@@ -1,16 +1,27 @@
 # ChatDemo-SwiftUI  
 
-Basic Chat application written with Ditto and SwiftUI/Combine  
+Basic Chat application written with Ditto and SwiftUI/Combine
 
-1. Clone this repo to a location on your machine, and open in Xcode    
-2. Navigate to the project Signing & Capabilities tab and modify the Team and Bundle Identifier 
+1. In the Ditto Portal, create a new app if you have not already done so.
+2. Clone this repo to a location on your machine, and open in Xcode    
+3. Navigate to the project Signing & Capabilities tab and modify the Team and Bundle Identifier 
 settings to your Apple developer account credentials to provision building to your device   
-3. In Terminal, run `cp .env.template .env` in the iOS directory  
-4. Edit `.env` to add environment variables as in the following example:  
+4. In Terminal, run `cp .env.template .env` in the iOS directory  
+
+5. Edit `.env` to add environment variables as in the following example, substituting your own values as needed (based on your app configuration in the portal you will only need a subset of these values):
 ```
-APP_ID=a01b2c34-5d6e-7fgh-ijkl-8mno9p0q12r3
-DITTO_OFFLINE_TOKEN=a01b2c34-5d6e-7fgh-ijkl-8mno9p0q12r3
+    DITTO_APP_ID = "replace with your app id"
+    DITTO_PLAYGROUND_TOKEN = "replace with your playground token"
+    DITTO_AUTH_PASSWORD = "replace with your auth password"
+    DITTO_AUTH_PROVIDER = "replace with your auth provider"
+    DITTO_OFFLINE_TOKEN = "replace with your offline license token"
```

Comment:
Similar comment regarding Android. Remove references to the offline token (in the readme and in the code) - I don't even see it in my DItto portal now.

Reply:
Based on the Android comments, it sounds like the offline token stays, but some tweaking to the language might be needed.

Reply:
I think we should remove the offline token statement from the README and the env.template. In cases of customers who get an offline token license, we can support them directly if needed, but we don't offer offline tokens for self serve customers.

Reply:
Based on this conversation, I've removed all references to the offline token in the Readme, config files, and code.